spec/datetimeformat.html

```diff
@@ -303,6 +305,20 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"millisecond"`, then
+            1. Let _msd_ be _dateTimeFormat_.[[FractionalSecondDigits]].
+            1. If _msd_ is greater than 0, then
+              1. Let _v_ be _tm_.[[millisecond]].
+              1. If _msd_ = 1, then
+                1. Let _v_ be floor(_v_ / 100).
+              1. Else if _msd_ = 2, then
+                1. Let _v_ be floor(_v_ / 10).
+              1. Let _nf3Options_ be ObjectCreate(*null*).
+              1. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"minimumIntegerDigits"`, _msd_).
+              1. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"useGrouping"`, *false*).
+              1. Let _nf3_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nf3Options_ &raquo;).
+              1. Let _fs_ be FormatNumber(_nf3_, _v_).
```

Comment:
Use `fv` (*formatted-value*) instead of `fs` for consistency with the rest of this abstract operation.